src/main.h

```diff
@@ -107,6 +107,8 @@ static const bool DEFAULT_TXINDEX = false;
 static const unsigned int DEFAULT_BANSCORE_THRESHOLD = 100;
 
 static const bool DEFAULT_TESTSAFEMODE = false;
+/** Default for -permitrbf */
+static const bool DEFAULT_PERMIT_REPLACEMENT = true;
```

Comment:
@gmaxwell it would only degrade users' ability to use it if node operators don't want to turn it on.

Since Bitcoin Core is sensitive to contentious changes, and since Opt-In RBF is contentious, I think the default should be the least contentious option for the sake of consistency.

For those politically minded, this would also contribute into a small way to the perception that Core is responsive to the desires of major businesses in the space.